src/test/java/UhrzeitenTest.java

```diff
@@ -18,23 +17,10 @@ public void assembleAllValidTimes() {
 		assertThat(times.size(), not(2000));
 	}
 
-	@Test(expected = IndexOutOfBoundsException.class)
+	@Test(expectedExceptions = IndexOutOfBoundsException.class)
 	public void testIndexOutOfBoundsException() {
 		Uhrzeiten uhrzeiten = new Uhrzeiten();
 		List<String> times = uhrzeiten.assembleAllValidTimes();
 		times.get(-1);
 	}
-
-	@Test
```

Comment:
Put this on top of our agenda, unfortunately I can not see directly what's going on with your Git client right now but it can't be that serious.